src/utils/file.ts

```diff
@@ -1,5 +1,43 @@
+import { guessMimeTypeFromFilename } from "@shapediver/viewer.utils.mime-type";
+
+/**
+ * Given a file name, extract the file extension. 
+ * @param fileName 
+ * @returns 
+ */
+export const getFileExtension = (fileName: string) => {
+	const match = fileName.match(/\.([0-9a-z]+)$/i);
+  
+	return (match && match[1]) ? match[1].toLowerCase() : undefined;
+};
+
 /**
- * Download a blob with and use the specified filename.
+ * Guess the mime type of a file by its extension.
+ * @param fileName 
+ * @returns 
+ */
+export const guessMimeTypeByExt = (fileName: string) => {
+	return guessMimeTypeFromFilename(fileName)[0];
+};
+
+/**
+ * In case a file is missing a mime type, try to guess it from the file name.
+ * @param file 
+ * @returns 
+ */
+export const guessMissingMimeType = (file: File | string) : File | string => {
```

Comment:
@snabela Should I add these helper functions to the mime-type package?

Reply:
@MajorMeerkatThe3rd yes this would be useful!

Reply:
@snabela I created a task for it: https://shapediver.atlassian.net/browse/SS-7657